### PR TITLE
Switch update-pr job in release brew from ubuntu to macos #140

### PR DESCRIPTION
**Pull request by ronakfof**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/workflows/run-differential-tests.yaml [hunk 1]
```diff
@@ -21,13 +21,23 @@ jobs:
...
+          curl -o litani.deb -L \
+            https://github.com/awslabs/aws-build-accumulator/releases/download/$litani_latest_release/litani-$litani_latest_release.deb
+          sudo apt-get update \
+            && sudo apt-get install --no-install-recommends --yes \
+                                                      ./cbmc.deb \
+                                                      ./litani.deb \
+                                                      ninja-build \
+                                                      universal-ctags
+          rm -f cbmc.deb litani.deb
           python3 -m pip install jinja2 cbmc-viewer
```
Comment (tautschnig) [inline .github/workflows/run-differential-tests.yaml:41]: It's perhaps too unrelated, but: why is `jinja2` not installed via `apt-get`?

Comment (ronakfof) [reply .github/workflows/run-differential-tests.yaml:41]: No preference. `Jinja2` being a pip package primarily, I installed it with `pip`. Can change if necessary.

**Comment by markrtuttle** [comment]
I suspect the ctags tests are failing because there was a code change to http that is yielding a line number change in symbol definitions.

**Comment by markrtuttle** [comment]
Closing in favor of https://github.com/model-checking/cbmc-viewer/pull/140

**Pull request closed without merging**